lib/ansible/plugins/strategy/__init__.py

```diff
@@ -514,11 +514,13 @@ def search_handler_blocks_by_name(handler_name, handler_blocks):
 
                             if handler_name in candidates:
                                 return handler_task
-                        except (UndefinedError, AnsibleUndefinedVariable):
+                        except (UndefinedError, AnsibleUndefinedVariable) as e:
                             # We skip this handler due to the fact that it may be using
                             # a variable in the name that was conditionally included via
                             # set_fact or some other method, and we don't want to error
                             # out unnecessarily
+                            if not handler_task.listen:
+                                display.warning("Unable to template handler name '%s': %s" % (handler_task.name, to_text(e)))
```

Comment:
Should we also mention that the handler won't be used or do you think it's implied by the current message?

Reply:
the handler is still usable, specially if 'listen' is present or they use the 'untemplated string' to call it

Reply:
@bcoca I think it is unusable, at least on devel, unless `listen` is present. Here's the last commit in this branch is what I needed to do to get it working https://github.com/ansible/ansible/compare/devel...s-hertel:pr/75244_plus_notify_unsafe_template_string?expand=1 and even then, the `notify` field needs to be unsafe or that will fail.

Reply:
I tested notifying with the untemplated string (with and without !unsafe) back to 2.6 and it doesn't appear to have worked on recent versions, though I didn't check anything older.